### DISCUSSION TITLE
502 Bad Gateway  Certificate verify failed: unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by gg4u** [mitmproxy/mitmproxy#5776] [Q&A]
#### Problem Description
I am accessing to a database licenced by the university.
Access is through a url like:
https://XXXXX.ezp.sub.UNIVERSITY.COM/version-20221117/XXXX

it successfully load if not using mitmproxy.
When activated:

502 Bad Gateway  Certificate verify failed: unable to get local issuer certificate

#### System Information
Version of mitmproxy is 8.1.1

--

Is mitmproxy blocked by the remote server ?

**Comment by kuan121** [comment]
See this [answer](https://github.com/mitmproxy/mitmproxy/issues/1608#issuecomment-347925027) for the github issue [Certificate Verification Error: unable to get local issuer certificate](https://github.com/mitmproxy/mitmproxy/issues/1608)
